packages/wonder-blocks-link/src/components/link-core.tsx

```diff
@@ -6,12 +6,15 @@ import {__RouterContext} from "react-router";
 import {addStyle} from "@khanacademy/wonder-blocks-core";
 import Color, {mix, fade} from "@khanacademy/wonder-blocks-color";
 import {isClientSideUrl} from "@khanacademy/wonder-blocks-clickable";
+import Icon from "@khanacademy/wonder-blocks-icon";
+import Spacing from "@khanacademy/wonder-blocks-spacing";
```

Comment:
These error messages mean that references for these packages need to be added to the tsconfig.json file for wonder-blocks-link.